lms/djangoapps/courseware/models.py

```diff
@@ -248,3 +251,56 @@ class Meta(object):   # pylint: disable=missing-docstring
 
     field = models.CharField(max_length=255)
     value = models.TextField(default='null')
+
+
+SCORE_CHANGED = Signal(
+    providing_args=[
+        'points_possible',  # Maximum score available for the exercise
+        'points_earned',   # Score obtained by the user
+        'user',  # User object representing the student
+        'course',  # CourseLocator object representing the course
+        'usage'  # BlockUsageLocator indicating the courseware instance
+    ]
+)
+
+
+@receiver(score_set)
+def submissions_score_set_handler(sender, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Herm, perhaps the docstring can both reflect the necessity for **kwargs, as well as explain what kwargs are expected?